improver_tests/nbhood/nbhood/test_GeneratePercentilesFromANeighbourhood.py

```diff
@@ -50,20 +52,8 @@ def test_neighbourhood_method_exists(self):
         Test that no exception is raised if the requested neighbourhood method
         exists.
         """
-        neighbourhood_method = "circular"
         radii = 10000
-        NBHood(neighbourhood_method, radii)
-
-    def test_neighbourhood_method_does_not_exist(self):
```

Comment:
No longer needed as we only use circular kernels for GeneratePercentilesFromANeighbourhood and it is no longer forced to share an interface with other plugins.